packages/Webkul/Shipping/src/Config/carriers.php

```diff
@@ -19,6 +20,7 @@
         'active'           => true,
         'is_calculate_tax' => true,
         'default_rate'     => '0',
+        'default'          => 1,
```

Comment:
In this file, both default key is set to 1. May I know how it is working? so that I can check further.

Reply:
Since, 'default' is a key and at most of times default makes sense to be only pre-defined selection in any document flow.
And due to some limitation in configuration we cannot place the global page rules on configuration yet. So we need to 
make sure that default should be set to true for one configuration only, till we can place document level validations.

As of now i have also did the same with existing shipment methods available.